zmq-async.opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "anders@fugmann.net"
+authors: ["Rudi Grinberg"]
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+dev-repo: "git@github.com/issuu/ocaml-zmq.git"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "zmq"
+  "jbuilder" {build}
+  "configurator" {build}
+  "ppx_sexp_conv" {build & >= "v0.9.0"}
+  "base" {>= "v0.10.0"}
+  "async_unix" {>= "v0.10.0"}
```

Comment:
I think the dependency on `async_kernel` should be stated here as well instead of assuming `async_unix` will pull it in.